lighty-core/lighty-codecs/src/main/java/io/lighty/codecs/SerializeIdentifierCodec.java

```diff
@@ -63,8 +63,10 @@ public YangInstanceIdentifier serialize(final String identifier) {
         DataSchemaContextNode<?> schemaNode = this.dataSchemaContextTree.getRoot();
         for (final String args : pathArgs) {
             final QName qName = getQname(qNameModule, args);
-            if (schemaNode != null && schemaNode.getChild(qName) != null && schemaNode.getChild(qName).isMixin()) {
+            if (schemaNode != null) {
                 schemaNode = schemaNode.getChild(qName);
+            }
+            if (schemaNode != null && schemaNode.isMixin()) {
```

Comment:
this is less readable
the class is deprecated - will be removed in next release

Reply:
ok, reverted